### PR TITLE
AFT: Change code to be compatible with Python 2 and 3

### PR DESCRIPTION
**Pull request by skuusela**
Signed-off-by: Simo Kuusela simo.kuusela@intel.com

### CODE REVIEW

File: devices/edisondevice.py [hunk 1]
```diff
@@ -311,8 +314,8 @@ def _add_ssh_key(self):
         os.chown(ssh_directory, 0, 0)
         os.chown(authorized_keys_file, 0, 0)
         # Note: incompatibility with Python 3 in chmod octal numbers
```
Comment (Valtis) [inline devices/edisondevice.py]: Comment is no longer true and can be removed.

File: devices/virtualboxdevice.py [hunk 2]
```diff
@@ -309,8 +312,8 @@ def _do_inject_ssh_key(self):
 
         # and ensure the permissions are correct
         # Note: incompatibility with Python 3 in chmod octal numbers
```
Comment (Valtis) [inline devices/virtualboxdevice.py]: The comment is no longer accurate here either.

File: setup.py [hunk 3]
```diff
@@ -32,6 +32,12 @@
 TEST_PLANS = [filename for filename in TEST_PLANS if CONFIG_FILTER(filename)]
 CONFIG_FILES =  [filename for filename in CONFIG_FILES if CONFIG_FILTER(filename)]
 
+#Depending on python version, depencies will differ
```
Comment (Valtis) [inline setup.py]: Typo

**Comment by skuusela** [comment]
@Valtis Fixed the things you pointed out.

**Comment by Valtis** [comment]
retest this please

**Pull request merged**